prediction_market_agent/agents/prophet_agent/deploy.py

```diff
@@ -64,6 +64,28 @@ def load(self) -> None:
         )
 
 
+class DeployablePredictionProphetGPT4ominiAgent(DeployableTraderAgentER):
+    bet_on_n_markets_per_run = 20
+    agent: PredictionProphetAgent
+
+    # TODO: Uncomment and configure after we get some historic bet data
+    # def get_betting_strategy(self, market: AgentMarket) -> BettingStrategy:
+    #     return KellyBettingStrategy(
+    #         max_bet_amount=get_maximum_possible_bet_amount(
+    #             min_=1, max_=5, trading_balance=market.get_trade_balance(APIKeys())
+    #         ),
+    #         max_price_impact=0.7,
+    #     )
```

Comment:
I seem to remember Kelly outperformed fixed amounts on all scenarios - any reason not to use Kelly here?

Reply:
It didn't; some agents lost money. We agreed to first deploy the agent with tiny bets, gather data, and then run the benchmark to figure out what strategy to use.

Reply:
ok